Cargo.toml

```diff
@@ -70,6 +70,7 @@ trycmd = "0.13.4"
 default = []
 managed = []
 with_crash_reporting = []
+profiling = ["sentry"]
```

Comment:
the SDK is still used for other things in sentry-cli. Only the profiling (and frame pointer) feature needs to be optional in this case.

Reply:
I think that `sentry` get imported in line `43` in the "normal" dependency.

This is more for the optional one at line `90` specific to Unix.

I can't use the `"package-name?/feature-name"` syntax to simplify this because as you said `sentry` as a dependency is already used regardless of profiling and I would end up adding profiling even when I should not.

Not sure if there is a better way to handle this case but I'm open for suggestions.